sdk/eventhub/Azure.Messaging.EventHubs/src/Compatibility/TrackOneExceptionExtensions.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Messaging.EventHubs.Core;
+
+namespace Azure.Messaging.EventHubs.Compatibility
+{
+    /// <summary>
+    ///   The set of extension methods for exception types defined in the
+    ///   Track One client library.
+    /// </summary>
+    ///
+    internal static class TrackOneExceptionExtensions
+    {
+        /// <summary>
+        ///   Maps a <see cref="TrackOne.EventHubsException" /> or a child of it to the equivilent
+        ///   exception for the new API surface.
+        /// </summary>
+        ///
+        /// <param name="instance">The instance that this method was invoked on.</param>
+        ///
+        /// <returns>The equivilent exeption type for the new API, wrapping the <paramref name="instance" /> as the inner exception.</returns>
+        ///
+        public static Errors.EventHubsException MapException(this TrackOne.EventHubsException instance)
```

Comment:
Not a huge fan of the name, but I struggled to come up with anything better.   Given the small scope of use and the short time that it'll be with us, I'm fine with settling for it, but am open to suggestions for a better one.

Reply:
`EquivalentErrorException` ? I'm not good with naming either

Reply:
Maybe `EquivalentTrackTwoException`, `EquivalentException` or something along those lines?

Reply:
Good suggestions, but C# convention calls for methods being named with a verb.   `MapToError`?    `TranslateToEquivilent`?   I'm trying to avoid `TrackTwo` but that may be silly, since we're intentionally using `TrackOne` to make people uncomfortable and ensure that it is obvious we need to remove it.

Reply:
I went with `MapToTrackTwoException`.    I hate it worse, but it is still short-term and I think it makes the intention more clear.   Thanks for the suggestions.